android/app/src/main/res/values-el/strings.xml

```diff
@@ -195,8 +195,12 @@
 	<string name="prefs_group_route">Πλοήγηση</string>
 	<string name="pref_zoom_title">Πλήκτρα μεγέθυνσης</string>
 	<string name="pref_zoom_summary">Εμφάνιση στο χάρτη</string>
-	<!-- Settings «Map» category: «Night style» title -->
-	<string name="pref_map_style_title">Νυχτερινή λειτουργία</string>
+	<!-- Settings «Map» category: «Appearance» title -->
+	<string name="pref_map_style_title">Εμφάνιση</string>
+	<!-- Settings "Appearance" category: "Light" title -->
+	<string name="pref_appearance_light">Ανοιχτόχρωμη</string>
```

Comment:
To be consistent between the two options, it should be

- Light: `Ανοιχτόχρωμη`
- Dark: `Σκουρόχρωμη`

or

- Light: `Ανοιχτή`
- Dark: `Σκούρα`